src/render/context/mod.rs

```diff
@@ -93,7 +101,7 @@ pub struct Context {
     pub follow_links: bool,
 
     /// Number of threads to use; defaults to number of logical cores available
-    #[arg(short, long, default_value_t = num_cpus::get())]
+    #[arg(short, long, default_value_t = { default_threads() })]
```

Comment:
Is the block expression necessary?

Reply:
Was lingering from before I split it into a function. Good catch :+1: